src/main/java/sirius/biz/analytics/scheduler/AnalyticsBatchExecutor.java

```diff
@@ -84,18 +84,18 @@ private void scheduleNextLevel(LocalDate date, int nextLevel, AnalyticsScheduler
             boolean noOtherAnalyticsTasksRunning =
                     namedRegions.isNamedRegionFree(determineRegionName(scheduler.getName()));
             if (timeout.isReached() || noOtherAnalyticsTasksRunning) {
```

Comment:
**Optional:** Would prefer an if-elseif-else chain for readability instead of a combined if with a nested if-else just to save duplicating a single line